### PR TITLE
79 gui integration

### LINKED ISSUE #79: GUI implementation
Use branch 'gui-integration' to make a gui using java swing/etc and PR for peer review.

### PR DESCRIPTION
**Pull request by kevvo15**
GUI implementation for our database program. As of now it runs when main.main() is called. It still needs work on the Update operation and more functionality in reports. Resolves #79

### CODE REVIEW

**Review by domschneider — APPROVED** [review]
I think this is great progress on a simple gui. Good job Kevin.

**Review by alexandersanford — APPROVED** [review]
This GUI looks wonderful. Having the same functionality as our menu but with clickable buttons will definitely improve the end-user experience. Thanks for working on this!

File: pom.xml [hunk 1]
```diff
@@ -164,5 +164,15 @@
 				</plugin>
 			</plugins>
 		</pluginManagement>
-  </build>
+        <plugins>
```
[the remark this answers is not in the document]
Comment (JonathanGrant92) [reply pom.xml:167]: We can actually use 9 though. Go to line 113 and 114, where a different version of this artifact is stated, and modify accordingly. No need for a duplicate of the same plugin!

**Review by JonathanGrant92 — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreamteam/database/GUI.java [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+                panel2.add(date);
+                panel2.add(submit);
+                panel2.add(label);
+                panel2.add(view);
+                newGUI("REPORTS");
+
+                break;
+
+            case "QUIT":
+                System.exit(0);
```
Comment (JonathanGrant92) [inline src/main/java/com/dreamteam/database/GUI.java:184]: Change to frame.dispose() and frame2.dispose() to release gui resources without forcing a shutdown of the program.

**Review by JonathanGrant92 — CHANGES_REQUESTED** [review]

**Review by JonathanGrant92 — APPROVED** [review]

**Review by Alejandro-Rojas — APPROVED** [review]

**Pull request merged**